### PR TITLE
Added support for nextCell and prevCell

### PR DESCRIPTION
**Pull request by ryang1428**

### CODE REVIEW

File: FXForms/Controllers/FXFormController.m [hunk 1]
```diff
@@ -258,12 +258,92 @@ - (NSIndexPath *)indexPathForNextCellAfterCell:(id<FXFormFieldCell>)cell {
...
+        else if([self numberOfSections] > (NSUInteger)(indexPath.section)) {
+            return [NSIndexPath indexPathForItem:0 inSection:indexPath.section - 1];
+        }
+    }
+    
+    return nil;
+}
+
+- (NSIndexPath *)indexPathForPreviousCellAfterCell:(id<FXFormFieldCell>)cell {
+
```
Comment (dbergman) [inline FXForms/Controllers/FXFormController.m]: space

File: FXForms/Controllers/FXFormController.m [hunk 2]
```diff
@@ -258,12 +258,92 @@ - (NSIndexPath *)indexPathForNextCellAfterCell:(id<FXFormFieldCell>)cell {
     }];
     
     if (indexPath) {
+        if ([self numberOfFieldsInSection:indexPath.section] > (NSUInteger)(indexPath.item + 1)) {
+            NSIndexPath *ip = [NSIndexPath indexPathForItem:indexPath.item + 1 inSection:indexPath.section];
+            FXFormBaseCell *nextCell = (FXFormBaseCell *)[self cellForRowAtIndexPath:ip];
+            NSLog(@"nextField: %@", nextCell.field.type);
+
```
Comment (flippinjoe) [inline FXForms/Controllers/FXFormController.m:264]: Remove the log

File: FXForms/Controllers/FXFormController.m [hunk 3]
```diff
@@ -258,12 +258,92 @@ - (NSIndexPath *)indexPathForNextCellAfterCell:(id<FXFormFieldCell>)cell {
...
-            return [NSIndexPath indexPathForItem:0 inSection:indexPath.section+1];
+            return [NSIndexPath indexPathForItem:0 inSection:indexPath.section + 1];
+        }
+    }
+    
+    return nil;
+}
+
+- (NSIndexPath *)indexPathForPreviousResponderCellAfterCell:(id<FXFormFieldCell>)cell {
+    __block NSIndexPath *indexPath = nil;
```
Comment (flippinjoe) [inline FXForms/Controllers/FXFormController.m:302]: Shouldn't this be `beforeCell`

File: FXForms/Controllers/FXFormController.m (hunk 1 above)
Comment (flippinjoe) [inline FXForms/Controllers/FXFormController.m]: Also `beforeCell`

File: FXForms/Controllers/FXFormController_Private.h [hunk 4]
```diff
@@ -30,6 +30,7 @@
 @property (nonatomic, strong) NSMutableDictionary *cellClassesForFieldClasses;
 @property (nonatomic, strong) NSMutableDictionary *controllerClassesForFieldTypes;
 @property (nonatomic, strong) NSMutableDictionary *controllerClassesForFieldClasses;
+@property (nonatomic, strong) id<FXFormFieldCell> currentResponderCell;
```
Comment (flippinjoe) [inline FXForms/Controllers/FXFormController_Private.h:34]: This probably should be weak

**Comment by flippinjoe** [comment]
Just some styling tweaks.   Overall I think this makes sense.

**Pull request merged**